enable-tls-between-components.md

```diff
@@ -141,50 +141,49 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
 
 > **注意：**
 >
-> 目前 PD 的 `cert-allowed-cn` 配置项只能设置一个值。因此所有认证对象的 `commonName` 都要设置成同一个值。
+> 8.3 及之前版本的 PD 的 `cert-allowed-cn` 配置项只能设置一个值。因此所有认证对象的 `Common Name` 都要设置成同一个值。8.3 及之前版本的例子参见[此前文档](https://docs.pingcap.com/zh/tidb/v8.3/enable-tls-between-components)
+> 8.4 及之后版本的 PD 支持多 `Common Name`，可以根据需要设置成多个值。
+> 需要额外注意的是，tiup 在查询组件状态的时候会使用独立的标识，比如集群名是 `test`，它会使用 `test-client`。
 
 - TiDB
 
-    在 `config` 文件或命令行参数中设置：
-
-    ```toml
-    [security]
-    cluster-verify-cn = ["TiDB"]
-    ```
+  在 `config` 文件或命令行参数中设置：
 
+  ```toml
+  [security]
+  cluster-verify-cn = ["tidb", "test-client", "prometheus"]
+  ```
 - TiKV
 
-    在 `config` 文件或命令行参数中设置：
-
-    ```toml
-    [security]
-    cert-allowed-cn = ["TiDB"]
-    ```
+  在 `config` 文件或命令行参数中设置：
 
+  ```toml
+  [security]
+  cert-allowed-cn = ["tidb", "pd", "tikv", "tiflash", "prometheus"]
```

Comment:
Why change to the lowercase?

Reply:
In the past, cn being the same was fine. Now we need to compare the name it's signed with to get it to work.

The cluster I tried to deploy directly with tiup used the lowercase